playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -29,12 +28,25 @@
       - strace
       - tcpdump
       etcd_ip: ETCD_IP_REPLACE
+      skip_node_svc_handlers: True
+      i_openshift_additional_repos_key: "{{ ansible_distribution }}-{{ openshift_azure_release }}"     
 
-  - name: add insights-client to package installs when on rhel
+  - name: add rhel specific variables
     set_fact:
       openshift_node_image_prep_packages: "{{ openshift_node_image_prep_packages | union(['insights-client']) }}"
+      openshift_additional_repos: "{{ openshift_additional_repos | union(azure_node_repos[i_openshift_additional_repos_key]) }}"
+    no_log: true
     when: openshift_deployment_type == 'openshift-enterprise'
 
+#  - name: set pre-release registry
```

Comment:
I think we can run this task only in master and ensure we remove it when we branch off to a release branch.

Reply:
so this was the intention. But we need to pass metadata to ansible which is master run. This was intention of `OPENSHIFT_RELEASE` in aos-jobs repo. So we can enable required repos and do some "pre-release tweaking"